src/pages/myGeotab/guides/concepts.tsx

```diff
@@ -956,7 +956,7 @@ results = {
     ]
 }`
             } />
-        <h2>API Client Support</h2>
+        <h2 id="api-client-support">API Client Support</h2> {/* This has an id because there's a link to this section in the Using in Javascript page */}
```

Comment:
I think this should be "API client support"

Reply:
This isn't my page, should be handled by Wajih.